packages/scroller/src/vaadin-scroller.js

```diff
@@ -110,6 +110,38 @@ class Scroller extends FocusMixin(ElementMixin(ControllerMixin(ThemableMixin(Pol
   _shouldSetFocus(event) {
     return event.target === this;
   }
+
+  /**
+   * Scrolls the textarea to the start if it has a vertical scrollbar.
+   */
+  scrollToStart() {
+    switch (this.scrollDirection) {
+      case 'vertical':
+        this.scrollTop = 0;
+        break;
+      case 'horizontal':
+        this.scrollLeft = 0;
+        break;
+      default:
+        break;
+    }
+  }
+
+  /**
+   * Scrolls the textarea to the end if it has a vertical or horizontal scrollbar.
+   */
+  scrollToEnd() {
+    switch (this.scrollDirection) {
+      case 'vertical':
+        this.scrollTop = this.scrollHeight;
+        break;
+      case 'horizontal':
+        this.scrollLeft = this.__isRTL ? -this.scrollWidth : this.scrollWidth;
```

Comment:
Could `setNormalizedScrollLeft` be used here?

Reply:
`setNormalizedScrollLeft` currently calculates incorrectly. I discussed this with Serhii and the conclusion was to leave `setNormalizedScrollLeft` as it is and just not use it here.

Reply:
Ok, I see. `setNormalizedScrollLeft` doesn't currently work as one might expect.